packages/grid/x-data-grid/src/models/gridFilterModel.ts

```diff
@@ -8,11 +8,21 @@ export interface GridFilterModel {
    * @default []
    */
   items: GridFilterItem[];
-
   /**
    * - `GridLinkOperator.And`: the row must pass all the filter items.
    * - `GridLinkOperator.Or`: the row must pass at least on filter item.
    * @default `GridLinkOperator.Or`
    */
   linkOperator?: GridLinkOperator;
+  /**
+   * values used to quick filter rows
+   * @default `[]`
+   */
+  quickFilterValues?: any[];
+  /**
+   * - `GridLinkOperator.And`: the row must pass all the values.
+   * - `GridLinkOperator.Or`: the row must pass at least one value.
+   * @default `GridLinkOperator.Or`
+   */
+  quickFilterLinkOperator?: GridLinkOperator;
```

Comment:
The "link operator" name is confusing. I remember we discussed about renaming to "logic operator". Wouldn't make sense to start using this new name here?

```suggestion
  quickFilterLogicOperator?: GridLinkOperator;
```

I found https://github.com/mui/mui-x/pull/3915#discussion_r806575519